WalletWasabi/WabiSabi/Models/RoundState.cs

```diff
@@ -13,16 +15,33 @@ namespace WalletWasabi.WabiSabi.Models
 		FeeRate FeeRate,
 		Phase Phase,
 		TimeSpan ConnectionConfirmationTimeout,
+		ulong MaxAmountPerAlice,
+		ulong PerAliceVsizeAllocation,
```

Comment:
nit: when I chose it I think "per alice vsize allocation" sounded more like english to me, but it should probably be renamed to be consistent since everything else puts `PerAlice` at the end?

Reply:
I would prefer something more like `MaxVsizePerAlice` just to make clear this and the `MaxAmountPerAlice` are the same but for different kind of credentials.

Reply:
Also those two different fields (max & allocation) are very different conceptually. For amount the allocation is implicitly 0 and the max value is very large. For size `MaxVsize` (which is per credential request, not per Alice) and this initial allocation of vsize units that every alice gets.

I don't see much practical value in keeping them distinct, we can force them to be equal, but conceptually I think it's confusing to do that. At the very least this should have a name that makes it clear that one starts from 0 and another starts from an arbitrary values, i.e. that this number is both the max value and the initial value for vsize.

Why not have:

```
ulong MaxAmountPerAlice
ulong MaxVsizePerAlice
ulong VsizeAllocationPerAlice
```

even if `MaxVsizePerAlice` and `VsizeAllocationPerAlice` are both always going to be the same, even if there is only one underlying configuration variable, it avoids the apparent confusion of using a constraint as an initial value.